include/SFML/Window/ContextSettings.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef SFML_CONTEXTSETTINGS_HPP
 #define SFML_CONTEXTSETTINGS_HPP
 
+#include <SFML/Config.hpp>
```

Comment:
Actually, now I remember: because I changed the order of some includes, this file no longer compiled (it's using `Uint32`). So this is actually needed and abides to our documented inclusion order.